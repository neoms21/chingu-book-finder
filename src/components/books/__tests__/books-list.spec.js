import { shallow } from 'enzyme';
import React from 'react';
import BooksList from '../books-list';

describe('Books lists tests', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<BooksList books={[{ id: 'a' }, { id: 'b' }]} />);
    expect(wrapper).toMatchSnapshot();
  });

  it('renders error message when no books', () => {
    const wrapper = shallow(<BooksList />);
    expect(wrapper).toMatchSnapshot();
  });
});
