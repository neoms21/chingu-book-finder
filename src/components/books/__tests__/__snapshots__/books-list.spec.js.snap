// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Books lists tests renders correctly 1`] = `
<styled.div>
  <Book
    book={
      Object {
        "id": "a",
      }
    }
    key="a"
  />
  <Book
    book={
      Object {
        "id": "b",
      }
    }
    key="b"
  />
</styled.div>
`;

exports[`Books lists tests renders error message when no books 1`] = `
<styled.div>
  <ErrorMessage
    message="No books for this query!!"
  />
</styled.div>
`;
